core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -491,25 +491,43 @@ namespace kagome::consensus::grandpa {
   }
 
   void GrandpaImpl::onFinalize(const libp2p::peer::PeerId &peer_id,
-                               const Fin &fin) {
-    if (not is_ready_) {
-      return;
-    }
-
-    if (fin.justification.round_number != fin.round_number) {
-      logger_->warn(
-          "Round does not correspond to the fin message it belongs to");
+                               const FullCommitMessage &fin) {
+    SL_DEBUG(logger_,
+             "Finalization has received from[{}] = {} for block[{}] = {}",
```

Comment:
Please, use uniformed way as in another places: **block #{}**, and **from {}** or from **peer_id={}**